mandatory/3-function-output.js

```diff
@@ -1,18 +1,23 @@
 // Add comments to explain what this function does. You're meant to use Google!
 function getRandomNumber() {
+  // this function will multiply 10 to a random number between .0 to 1
   return Math.random() * 10;
 }
 
 // Add comments to explain what this function does. You're meant to use Google!
 function combine2Words(word1, word2) {
+// this function will contac 2 arguments passed
   return word1.concat(word2);
 }
 
+
 function concatenate(firstWord, secondWord, thirdWord) {
+  return `${firstWord} ${secondWord} ${thirdWord}`
```

Comment:
Very good understanding of interpolation but in this exercise we are being asked to use  the concat() method. Remember concat() doesn't  add spaces between strings in this case we have to use the   " " to add space.